src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -452,27 +475,16 @@
      * @param <T> expected inventory type, specified as generic to allow easy
      *      pseudo-duck-typing
      * @return the query result
-     */
-    <T extends Inventory> T query(String... names);
-
-    /**
-     * <p>Query this inventory by dynamically inspecting each operand. Each
-     * operand in turn is checked for a match against the other query methods,
-     * and if a matching method is found the query is performed using the
-     * operand. This is repeated until all operands are consumed and allows a
-     * union of multiple query types to be aggregated into a single view.</p>
-     *
-     * <p>For operands with no matching type, the behaviour is determined by the
-     * individual inventory. A naive match may be obtained by calling .equals()
-     * against the child inventory passing the unknown operand as an argument.
-     * </p>
-     *
-     * @param args search parameters
-     * @param <T> expected inventory type, specified as generic to allow easy
-     *      pseudo-duck-typing
-     * @return the query result
-     */
-    <T extends Inventory> T query(Object... args);
```

Comment:
Probably to `/dev/null`, which seems to be a very broken peripheral hidden inside my computer somewhere - I can't even read from it. Let me see if I can get it replaced with something.